modules/cudn-status-conditions.adoc

```diff
@@ -80,3 +80,157 @@ h|Message
+|You must set the `spec.network.localnet.subnets` field when using the `spec.network.localnet.excludeSubnet` field.
+
+|The CIDR range is invalid.
+|`The ClusterUserDefinedNetwork "localnet-subnets-invalid-ipv4-cidr-fail" is invalid: spec.network.localnet.subnets[0]: Invalid value: "string": CIDR is invalid`
+|You must set an acceptable CIDR range for `spec.network.localnet.subnets` field. Acceptable values are IPv4 and IPv6 CIDR ranges which are not in use or reserved by {product-title}.
+
+|You must set the `subnets` field when the `ipam.mode` is `Enabled` or when the IPAM mode is unset because the default value is `Enabled`.
+|`Subnets is required with ipam.mode is Enabled or unset, and forbidden otherwise`.
+|You must set the `spec.network.localnet.subnets` field unless the `spec.network.localnet.ipam.mode` is explicitly disabled.
+
+|Setting two CIDR ranges for `spec.network.localnet.subnets` field requires that one be IPv4 and the other be IPv6.
+|`Invalid value...When 2 CIDRs are set, they must be from different IP families`.
+|You must change one of your CIDR ranges to a different IP family.
+
+|The `spec.network.localnet.ipam.mode` is `Disabled` but the ``spec.network.localnet.lifecycle` has a value of `Persistent`.
+|`lifecycle Persistent is only supported when ipam.mode is Enabled`
+|You must set the `ipam.mode` to `Enabled` when the optional field `lifecycle` has a value of `Persistent`.
+|===
+
+.Invalid `vlan` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`invalid vlan` or `invalid mode`
+3+|One of the following messages is returned when the `spec.network.localnet.vlan` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The `spec.network.localnet.vlan.mode` field must be set.
+|`spec.network.localnet.vlan.mode: Unsupported value: "Disabled": supported values: "Access`
+|You must set the `spec.network.localnet.vlan.mode` field to `Access` mode.
+
+|The `spec.network.localnet.vlan` field must be set when `spec.network.localnet.vlan.mode` is set to `Access` mode.
+|`vlan access config is required when vlan mode is 'Access', and forbidden otherwise`.
+|You must set `spec.network.localnet.vlan.mode.access` field when using `Access` mode.
+
+|The `spec.network.localnet.vlan.access.id` value must be set when using `Access` mode.
+|`spec.network.localnet.vlan.access.id: Required value`
+|You must set a value for `spec.network.localnet.mode.access.id`.
+
+|Acceptable values for `access.id` are greater than or equal to 1.
+|`spec.network.localnet.vlan.access.id in body should be greater than or equal to 1`
+|You must set a value of 1 or greater for `access.id` field.
+
+|Acceptable values for `access.id` are less than or equal to 4094.
+|`spec.network.localnet.vlan.access.id in body should be less than or equal to 4094`
+|You must set a value of 4094 or less for `access.id` field.
```

Comment:
for whatever reason, I still don't see this in the preview ...

Reply:
I don't see this in the preview either. I think that line 211 needs changed to .7+ but I'd have to mess around with it.

Reply:
done